pkg/scheduler/core/extender.go

```diff
@@ -125,8 +125,8 @@ func (h *HTTPExtender) IsIgnorable() bool {
 	return h.ignorable
 }
 
-// SupportsPreemption returns if a extender support preemption.
-// A extender should have preempt verb defined and enabled its own node cache.
+// SupportsPreemption returns true if an extender support preemption.
```

Comment:
there is one more grammatical error in this comment:
s/support/supports/

Reply:
i‘ve replaced it